packages/core/demo/data/bar.ts

```diff
@@ -92,8 +92,20 @@ export const simpleBarLongLabelOptions = {
 		bottom: {
 			mapsTo: "group",
 			scaleType: "labels",
+			"truncation": {
+				"type": "TruncationTypes.END_LINE",
```

Comment:
```suggestion
				"type": TruncationTypes.END_LINE,
```

Reply:
These need to be turned into ENUMs, see this line please:
https://github.com/carbon-design-system/carbon-charts/blob/master/packages/core/src/interfaces/enums.ts#L17